packages/admin-stories/src/admin/table/TableWithFilterBar.tsx

```diff
@@ -0,0 +1,160 @@
+import { Field, FilterBar, FinalFormInput, Table, TableFilterFinalForm, TableQuery, useTableQuery, useTableQueryFilter } from "@comet/admin";
+import { FinalFormReactSelectStaticOptions } from "@comet/admin-react-select";
+import { Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import gql from "graphql-tag";
+import * as qs from "qs";
+import * as React from "react";
+
+import { apolloStoryDecorator } from "../../apollo-story.decorator";
+
+const gqlRest = gql;
+
+const query = gqlRest`
+query users(
+    $pathFunction: any
+    $username: String
+    $name: String
+    $email: String
+    $website: String
+) {
+    users(
+        email: $email
+        username: $username
+        name: $name
+        website: $website
+    ) @rest(type: "User", pathBuilder: $pathFunction) {
+        id
+        name
+        username
+        email
+        phone
+        website
+    }
+}
+`;
+function pathFunction({ args }: { args: { [key: string]: any } }) {
+    interface IPathMapping {
+        [arg: string]: string;
+    }
+    const paramMapping: IPathMapping = {
+        email: "email",
+        username: "username",
+        name: "name",
+        website: "website",
+    };
+
+    const q = Object.keys(args).reduce((acc: { [key: string]: any }, key: string): { [key: string]: any } => {
+        if (paramMapping[key] && args[key]) {
+            acc[paramMapping[key]] = args[key];
+        }
+        return acc;
+    }, {});
+
+    return `users?${qs.stringify(q, { arrayFormat: "brackets" })}`;
+}
+
+interface IQueryData {
+    users: Array<{
+        id: number;
+        name: string;
+        username: string;
+        email: string;
+        phone: string;
+        website: string;
+    }>;
+}
+
+interface IFilterValues {
+    username: any;
+    name: any;
+    email: any;
+    website: any;
+}
+
+interface IVariables extends IFilterValues {
+    pathFunction: any;
+}
+
+const ExampleWithSelect: React.FC = () => {
+    const options = [
+        { value: "Sincere@april.biz", label: "Sincere@april.biz" },
+        { value: "Shanna@melissa.tv", label: "Shanna@melissa.tv" },
+        { value: "Nathan@yesenia.net", label: "Nathan@yesenia.net" },
```

Comment:
could you use a better example for a select filter, like color or brand.

If you change the story to not use an API you can also change all other filters to make more sense.

Reply:
i used those values because in the remote source i do not have brand or color as a data column 
and i focused on the work of the filterbar component itself and i know the story needs to be better. in general i guess our stories need to be overworked, many of them have also some problems with it :)

Reply:
Please do not use bad stories as excuse for more bad stories!

Reply:
i dont not use that as an excuse! i was just honest and told the truth on how i did it !

Reply:
and how should i know that it was a bad story if it is in the storybook ? i just took it as an example for my own ! feeling offended right here tbh!

Reply:
Sorry, I really didn't want to offend you :( now I'm feeling bad....

Anyhow, we will sort this out today